lib/features/history/widgets/history_app_bar.dart

```diff
@@ -0,0 +1,29 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:givt_app_kids/shared/widgets/heading_2.dart';
+import 'package:givt_app_kids/shared/widgets/back_button.dart'
+    as custom_widgets;
+
+class HistoryAppBar extends StatelessWidget implements PreferredSizeWidget {
+  const HistoryAppBar({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    return AppBar(
+      backgroundColor: const Color(0xFFEEEDE4),
+      elevation: 0,
+      title: const Row(
+        children: [
+          Heading2(text: 'My Givts'),
+          Spacer(),
+        ],
+      ),
+      leading: custom_widgets.BackButton(
+        icon: SvgPicture.asset('assets/images/back_btn.svg'),
```

Comment:
I'm not sure that we need to customise icon in this case. Let's keep consistency from UX/UI perspective